testing/run_tests.py

```diff
@@ -155,10 +158,9 @@ def SnapshotTest(build_dir, dart_file, kernel_file_output, verbose_dart_snapshot
   ]
 
   if verbose_dart_snapshot:
-    subprocess.check_output(snapshot_command, cwd=buildroot_dir)
+    RunCmd(snapshot_command, cwd=buildroot_dir)
   else:
-    with open(os.devnull,"w") as out_file:
-      subprocess.check_output(snapshot_command, cwd=buildroot_dir, stdout=out_file)
+    subprocess.check_output(snapshot_command, cwd=buildroot_dir)
```

Comment:
nit: RunCmd here too?

Reply:
This one is supposed to be silent.  Just using check_output so we print the output if there's an error.